modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/TargetHandler.java

```diff
@@ -390,6 +415,17 @@ private boolean handle202(MessageContext requestMsgContext) throws AxisFault {
         return false;
     }
 
+    private void dropTargetConnection(NHttpClientConnection conn) {
```

Comment:
having a nice title to PR is good (as per guidelines set by git training done few weeks back)